### PR TITLE
#1-4Task. AleksanderGogolushko

### PR DESCRIPTION
**Pull request by AleksandrGogolushko**
https://travis-ci.org/github/AleksandrGogolushko/js-assignments

### CODE REVIEW

File: task/01-strings-tasks.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ function removeFirstOccurrences(str, value) {
  *   '<a>' => 'a'
  */
 function unbracketTag(str) {
-    throw new Error('Not implemented');
+    return str.replace(/^<|>$/mg, "")
 }
```
Comment (IlyaMokin) [inline task/01-strings-tasks.js:149]: !

File: task/01-strings-tasks.js [hunk 2]
```diff
@@ -221,9 +226,25 @@ function getRectangleString(width, height) {
...
+        letter = letter + 13 >= 26
+            ? alphabet[letter - 13]
+            : alphabet[letter + 13];
 
+        str[i] == strLowerCase[i]
+            ? result.push(letter)
+            : result.push(letter.toUpperCase())
+    }
+    return result.join("")
+}
```
Comment (IlyaMokin) [inline task/01-strings-tasks.js:247]: looks good, but can be easier

File: test/03-date-tests.js [hunk 3]
```diff
@@ -87,6 +87,14 @@ describe('03-date-tasks', function() {
...
                 expected:  '29:20:10.453'
+            },{
+                startDate: new Date(2000,1,1,10,0,0),
+                endDate:   new Date(2000,2,2,15,20,10,453),
+                expected:  '725:20:10.453'
+            },{
+                startDate: new Date(2000,1,1,10,0,0),
+                endDate:   new Date(2001,1,2,15,20,10,453),
+                expected:  '8813:20:10.453'
             }
```
Comment (IlyaMokin) [inline test/03-date-tests.js:98]: Why did you add additional test cases? is it really important?

Comment (AleksandrGogolushko) [reply test/03-date-tests.js:98]: I just wanted to check, and forgot to delete it later.
